src/streams/StreamItem.js

```diff
@@ -151,13 +164,26 @@ export default function StreamItem(props: Props): Node {
       }}
     >
       <View style={wrapperStyle}>
+        {handleExpandCollapse && (
+          <Pressable
+            style={componentStyles.touchable}
+            onPress={handleExpandCollapse && (() => handleExpandCollapse(streamId))}
```

Comment:
This `handleExpandCollapse &&` is happily now redundant with the one above. That simplification is one of the benefits of unifying that prop with the `isCollapsible` of the previous revision.

Reply:
@gnprice But don't we still need this conditional rendering of the `Pressable caret` since `StreamItem` gets called a few other places in the code. If thats taken away then wouldn't the `caret` always get rendered with `StreamItem`, which may not be preferred?

Here is what the second tab(screens) looks like if I remove that conditional. 

<img width="320" alt="Screen Shot 2022-09-08 at 8 51 13 AM" src="https://user-images.githubusercontent.com/50177345/189167980-38f1d23d-2309-464c-82e7-ae74e0ef915c.png">

Reply:
Right, I mean that because the first conditional (around the whole `Pressable`) is there, the conditional on this line (in the value of `onPress`) is redundant.

Reply:
Ahh oops I missed that! That makes more sense haha. Thanks for the clarification!